apps/client/ui/Range/Range.tsx

```diff
@@ -0,0 +1,101 @@
+  className?: string
+};
+
+const sliderStyles = ''
+
+export const CustomRange: FC<CustomRangeProps> = ({
+  min = 0,
+  max = 10,
+  step = 1,
+  defaultValue = [0, 10],
+  className,
+  ...props
+}) => {
+  const [values, setValues] = useState<number[]>(defaultValue);
+  const containerRef = useRef<HTMLDivElement>(null);
+
+  const handleOnChange = (newValues: number[]) => {
+    setValues(newValues);
+  };
+
+  const handleTooltipPosition = () => {
+    if (!containerRef.current) return;
+
+    const [leftHandle, rightHandle] = Array.from(containerRef.current.getElementsByClassName('rc-slider-handle')) as HTMLElement[];
+    const [leftTooltip, rightTooltip] = Array.from(containerRef.current.getElementsByClassName('slider-tooltip')) as HTMLElement[];
+
+    if (leftHandle && leftTooltip && rightHandle && rightTooltip) {
+      const containerRect = containerRef.current.getBoundingClientRect();
+      const leftHandleRect = leftHandle.getBoundingClientRect();
+      const rightHandleRect = rightHandle.getBoundingClientRect();
+      const rightTooltipRect = rightTooltip.getBoundingClientRect();
+
+      const leftOffset = leftHandleRect.left - containerRect.left - leftHandleRect.width;
+      leftTooltip.style.left = `${leftOffset}px`;
+
+      const rightOffset = rightHandleRect.left - containerRect.left + rightHandleRect.width - rightTooltipRect.width;
+      rightTooltip.style.left = `${rightOffset}px`;
+    }
+  };
+
+  useEffect(() => {
+    handleTooltipPosition();
+    window.addEventListener('resize', handleTooltipPosition);
+    return () => {
+      window.removeEventListener('resize', handleTooltipPosition);
+    };
+  }, [values]);
+
+  return (
+    <div className='min-w-[180px] mt-12 mx-6 mb-0 h-9' ref={containerRef} {...props}>
+      <Slider
+        range
+        min={min}
+        max={max}
+        step={step}
+        allowCross={false}
+        defaultValue={values}
+        onChange={handleOnChange}
+        trackStyle={[{ backgroundColor: 'var(--primary-color)', height: '4px' }]}
+        railStyle={{ backgroundColor: 'var(--dark-grey)', height: '4px' }}
+        handleStyle={[
+          {
+            backgroundColor: 'var(--primary-color)',
+            height: '14px',
+            width: '14px',
+            border: 'none',
+          },
+          {
+            backgroundColor: 'var(--primary-color)',
+            height: '14px',
+            width: '14px',
+            border: 'none',
+          },
+        ]}
+        className='w-full p-10'
+      />
+      <div className="flex mt-[5px] justify-between">
+        <div className="flex-1 text-left ml-[-5px] text-sm text-gray-500">{min}</div>
+        <div className="flex-1 text-right mr-[-7px] text-sm text-gray-500">{max}</div>
+      </div>
+      <div className='relative bg-dark-grey py-1.5 px-2 border-0 rounded text-xs z-[1] pointer-events-none text-text-color w-[45px] text-center shadow shadow-[#141417] mt-[-75px] ml-[-1px] slider-tooltip left-tooltip'>
```

Comment:
`clxs` would be helpfull to manage such long class strings like this 😎 you can group them to shorter strings and pass as arguments of `clsx` function. then they will break to next lines and will be easier to read 👌 There is some tip how to think of grouping classes - https://tailwindcss.com/blog/automatic-class-sorting-with-prettier#how-classes-are-sorted